projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/secrets/service/vault/VaultJobSecretsService.java

```diff
@@ -23,6 +27,9 @@
 @ConditionalOnProperty(value = "featureflag.vault.integration.enabled")
 public class VaultJobSecretsService implements com.vmware.taurus.secrets.service.JobSecretsService {
 
+  @Value("${vdk.vault.size.limit}")
+  private int sizeLimit = 1048576;
```

Comment:
The purist in me must mention : Avoid "magic numbers": The number 1048576 is hard-coded into the sizeLimit variable. Better it would be constant VAULT_SIZE_LIMIT_DEFAULT ...

Reply:
fixed.